docs/4.operation/3.fi-health.md

```diff
@@ -0,0 +1,28 @@
+## FI-Health
+
+FI-Health is an end-to-end testing tool that ensures that the OpenStack
+integrated components is working as expected. Each of the FIWARE Lab
+node is tested in a real-world scenario using the OpenStack Rest API
+provided by the federated Keystone service endpoints. The idea behind
+these tests is to provide end-to-end testing of the FIWARE Cloud Portal
+to test whether the flow of the application is performing as designed
+from start to finish. The purpose of carrying out these end-to-end tests
+is to identify OpenStack services problems before or at the same time
+that the users can observe those problems in the FIWARE Cloud Portal.
+Therefore, FI-Health is the key component to provide quality of service
+in the FIWARE Cloud environment.
+
+![Sanity Check tool](image6.png)
+
+You can obtain more information about the component by accessing the
+[*FIWARE Health GitHub
+repository*](https://github.com/telefonicaid/fiware-health/blob/develop/README.rst).
+Note that if you want to increase or improve this tool you can suggest
+new issues in this GitHub repository.
```

Comment:
new issues in this GitHub repository or even better provide us new PRs to extend the functionality of the service.